pymongo/client_session.py

```diff
@@ -827,6 +827,9 @@ def _start_retryable_write(self):
         self._check_ended()
         self._server_session.inc_transaction_id()
 
+    def __copy__(self):
+        raise RuntimeError('A session cannot be copied (simply)!')
```

Comment:
I think [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError) would be a better fit here:
> This exception may be raised by user code to indicate that an attempted operation on an object is not supported, and is not meant to be.

Also, can you change the message to:
```
'A ClientSession cannot be copied, create a new session instead'
```

Reply:
Thanks for the feedback... Done!